### PR TITLE
Adding example pages for components that didn't have one

### PR DESCRIPTION
**Pull request by williamw2**
# Pull Request

## 📖 Description

Adding example pages for the color picker and file components and updating the HTML Render example to include the Inline Edit component.
The File component example includes the file-action-objurl component as the file-action is more of an interface definition and components implementing it are not intended for use outside of a File component. Additional File example routes may be added when new file-action components are added. 
The HTML Render example button has been moved and re-labeled and updated to include the html-render-layer-inline-edit component. The html-render-layer, html-render-layer-inline-edit and html-render-layer-navigation components are not intended to be used outside of html-render and are included in the html-render route.
I also re-ordered the buttons in the web component section of the app/index.html to be ordered alphabetically. 

### 🎫 Issues

Closes: #96 

### General

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.

### CODE REVIEW

File: change/@microsoft-fast-tooling-66cace99-f127-40a7-9904-ed017826d40c.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```
Comment (janechu) [inline change/@microsoft-fast-tooling-66cace99-f127-40a7-9904-ed017826d40c.json:2]:
```suggestion
  "type": "none",
```

File: change/@microsoft-fast-tooling-66cace99-f127-40a7-9904-ed017826d40c.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  "comment": "Added example pages for components that didn't have one.",
+  "packageName": "@microsoft/fast-tooling",
+  "email": "44823142+williamw2@users.noreply.github.com",
+  "dependentChangeType": "patch"
```
Comment (janechu) [inline change/@microsoft-fast-tooling-66cace99-f127-40a7-9904-ed017826d40c.json:6]:
```suggestion
  "dependentChangeType": "none"
```

**Review by janechu — CHANGES_REQUESTED** [review]

**Review by janechu — APPROVED** [review]

File: change/@microsoft-fast-tooling-66cace99-f127-40a7-9904-ed017826d40c.json (hunk 1 above)
Comment (williamw2) [reply change/@microsoft-fast-tooling-66cace99-f127-40a7-9904-ed017826d40c.json]: Done

File: change/@microsoft-fast-tooling-66cace99-f127-40a7-9904-ed017826d40c.json (hunk 2 above)
Comment (williamw2) [reply change/@microsoft-fast-tooling-66cace99-f127-40a7-9904-ed017826d40c.json]: Done

**Pull request merged**